### PR TITLE
docs(webpack5): add new cache options

### PR DESCRIPTION
**Pull request by rishabh3112**
_describe your changes..._

Part of #2774
Adds documentations for:
1. `cache.idleTimeout` option
2. `cache.idleTimeoutForInitialStore` option

### CODE REVIEW

File: src/content/configuration/other-options.md [hunk 1]
```diff
@@ -210,6 +211,43 @@ module.exports = {
...
+```javascript
+module.exports = {
+  //..
+  cache: {
+    idleTimeout: 10000
+  }
+};
+```
+
+W> `cache.idleTimeout` is only available for when [`cache.store`](#cachestore) is set to either `'pack'` or `'idle'`
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md]: Lets use simpler form
```suggestion
W> `cache.idleTimeout` is only available when [`cache.store`](#cachestore) is set to either `'pack'` or `'idle'`
```

File: src/content/configuration/other-options.md [hunk 2]
```diff
@@ -210,6 +211,43 @@ module.exports = {
 
 W> Don't share the cache between calls with different options.
 
+### `cache.idleTimeout`
+
+`number = 10000`
+
+Time in `ms`. `cache.idleTimeout` denotes the time period after which the cache storing should happen. Defaults to `10000` which is equivalent to `10s`
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md]: No need to elaborate when it already says milliseconds
```suggestion
Time in milliseconds. `cache.idleTimeout` denotes the time period after which the cache storing should happen.
```

File: src/content/configuration/other-options.md [hunk 3]
```diff
@@ -210,6 +211,43 @@ module.exports = {
...
+};
+```
+
+W> `cache.idleTimeout` is only available for when [`cache.store`](#cachestore) is set to either `'pack'` or `'idle'`
+
+### `cache.idleTimeoutForInitialStore`
+
+`number = 0`
+
+Time in `ms`. `cache.idleTimeoutForInitialStore` is the time period after which the initial cache storing should happen. It defaults to `0` which is equivalent to `0s`.
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md]:
```suggestion
Time in milliseconds. `cache.idleTimeoutForInitialStore` is the time period after which the initial cache storing should happen.
```

File: src/content/configuration/other-options.md [hunk 4]
```diff
@@ -210,6 +211,43 @@ module.exports = {
...
+```javascript
+module.exports = {
+  //..
+  cache: {
+    idleTimoutForInitialStore: 0
+  }
+};
+```
+
+W> `cache.idleTimeoutForInitialStore` is only available for when [`cache.store`](#cachestore) is set to either `'pack'` or `'idle'`
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md]:
```suggestion
W> `cache.idleTimeoutForInitialStore` is only available when [`cache.store`](#cachestore) is set to either `'pack'` or `'idle'`
```

**Comment by rishabh3112** [comment]
Thanks @EugeneHlushko, Have applied your suggestions and it looks better now to me :)

**Comment by EugeneHlushko** [comment]
Thanks! Looks good

**Pull request merged**